certified-connectors/Tesseron ASM Ticket/apiDefinition.swagger.json

```diff
@@ -0,0 +1,819 @@
+                },
+                "delegatedTicketEditor": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Enter the editor of the ticket",
+                  "title": "Ticket Editor"
+                },
+                "enterpriseId": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Specify the ID of the business partner",
+                  "title": "Enterprise ID"
+                },
+                "contactId": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Select the main contact of the ticket",
+                  "title": "Contact ID"
+                }
+              },
+              "required": [
+                "areaId",
+                "effectsType",
+                "enterpriseId",
+                "entranceType",
+                "internalOption",
+                "privateOption",
+                "releasedOption",
+                "ticketHeader",
+                "ticketText",
+                "ticketType",
+                "urgencyType"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/CreateTicketPosition": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default"
+          },
+          "200": {
+            "description": "sucess",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "Message": {
+                  "type": "string",
+                  "description": "Message"
+                },
+                "Success": {
+                  "type": "boolean",
+                  "description": "Success"
+                },
+                "TicketNumber": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "description": "Ticket number of the edited ticket",
+                    "title": "Ticket Number"
+                  },
+                  "description": "TicketNumber"
+                },
+                "TicketPositionNumber": {
+                  "type": "string",
+                  "description": "ID of the created ticket position",
+                  "title": "Ticket Position ID"
+                }
+              }
+            },
+            "headers": {
+              "{": {
+                "description": "{",
+                "type": "string"
+              },
+              "\"cache-control\"": {
+                "description": "\"cache-control\"",
+                "type": "string"
+              },
+              "\"content-length\"": {
+                "description": "\"content-length\"",
+                "type": "string"
+              },
+              "\"content-security-policy\"": {
+                "description": "\"content-security-policy\"",
+                "type": "string"
```

Comment:
@developmentLL could you please tell me what are these headers for? How are you going to use them?